README_GAMELIFT.md

```diff
@@ -2,39 +2,110 @@
 
 This README covers testing and running MultiplayerSample with Amazon GameLift.
 
-## Local testing with [GameLiftLocal](https://docs.aws.amazon.com/gamelift/latest/developerguide/integration-testing-local.html)
+## Running with [GameLift](https://docs.aws.amazon.com/gamelift/index.html)
 
 ### Setup
-
 1. Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), if you don't already have it. You will need it to create a game session with GameLiftLocal.
-1. Download Java Runtime Environment: https://www.java.com/en/download/manual.jsp
-1. Download the [AmazonGameLift Server SDK](https://gamelift-release.s3-us-west-2.amazonaws.com/GameLift_06_03_2021.zip) and extract the "GameLiftLocal-1.0.5" directory somewhere easy to find.
 1. Enable the "AWSGameLift" and "MPSGameLift" gem by adding them to MultiplayerSample/Gem/Code/enabled_gems.cmake
 1. Build the server and game launchers for MultiplayerSample as normal, per [top-level README](/README.md).
+1. Work in progress (WiP) step: Add your AWS region to Config/default_aws_resource_mappings.json (example: "Region": "us-west-2")
+    a. Currently needed otherwise when the client initializes GameLift there will be an error about not having a region. 
+    b. This step will be removed once we properly parse the game-session data which contains the fleet-id, region-id, etc  
 
+## Build Server for Windows (WiP)
+1. WiP Step: Build a profile pak server/game build
```

Comment:
Can link to documentation here: https://github.com/o3de/o3de-multiplayersample/pull/321